iam_ltk.go

```diff
@@ -100,6 +106,37 @@ func (c *Client) GetLongTermKeys() (*GetLongTermKeysResponse, error) {
 	return cr, nil
 }
 
+// GetLongTermKey gets a single LTK for an account
+// If no error is returned, then you will receive an LTK for the given account.
+func (c *Client) GetLongTermKey(iamUsername string) (*GetLongTermKeyResponse, error) {
+	log.Printf("[INFO] Getting long term key")
+
+	req, err := c.NewRequest(nil, "GET", "/ltk/search/"+iamUsername)
```

Comment:
So I didn't get to review you ALKS Core PR, but how is the account name being communicated?  Right now this endpoint looks like it searches every account for a matching LTK.